### PR TITLE
Countervalues fixes

### PR DESCRIPTION
**Pull request by gre**
Fixes for countervalues

### CODE REVIEW

File: src/renderer/init.js [hunk 1]
```diff
@@ -113,8 +113,9 @@ async function init() {
   } else {
     store.dispatch(lock());
   }
+  const initialCountervalues = await getKey("app", "countervalues");
```
Comment (JunichiSugiura) [inline src/renderer/init.js:116]: 👍

**Review by JunichiSugiura — APPROVED** [review]
LGTM

File: src/renderer/ReactRoot.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ type State = {
 type Props = {
   store: Store<StoreState, *>,
   language: string,
+  initialCountervalues: *,
```
Comment (JunichiSugiura) [inline src/renderer/ReactRoot.js:15]: type CounterValuesStateRaw isn't it?

**Review by valpinkman — APPROVED** [review]

File: src/renderer/ReactRoot.js (hunk 2 above)
Comment (gre) [reply src/renderer/ReactRoot.js:15]: not easy to type i think. because it's a JSON from the store anyway.

**Pull request merged**